### PR TITLE
Blog App: validations and model specs - Day 3

### PR DESCRIPTION
**Pull request by hajnaloltyan**
## Blog App: validations and model specs - Day 3

In this PR, I:

* Installed RSpec for the Rails project
* Added the following validations for the User model:
        - Name must not be blank.
        - PostsCounter must be an integer greater than or equal to zero.
* and for the Post model:
        - Title must not be blank.
        - Title must not exceed 250 characters.
        - CommentsCounter must be an integer greater than or equal to zero.
        - LikesCounter must be an integer greater than or equal to zero.
 
* Added unit specs for all the models' methods and validations.
* Fixed linter

### CODE REVIEW

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
```
Comment (V-Blaze) [inline spec/models/user_spec.rb:3]: - You have done great so far :+1: But there are still a few requirements you need to meet before moving to the next milestone. :+1: :100: 

- - Screenshot reference:
Selection_019

- [x] As stated in the project requirement, you need to have a `unit specs for all of your models' methods`. Currently in your `users model spec` you do not have a test for the `most_recent_posts` method. I'll suggest you add that. :smile: :wave:

File: spec/models/comment_spec.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
```
Comment (V-Blaze) [inline spec/models/comment_spec.rb:3]: - [x] Just like I pointed out above, you also do not have a test case for the `comments_counter_updater` method. I'll suggest you add that too. :+1: :wave:

File: spec/models/like_spec.rb [hunk 3]
```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  let(:user) do
```
Comment (V-Blaze) [inline spec/models/like_spec.rb:4]: - [x] Also as I pointed out above, you need to add a test for the `likes_counter_updater` method in the likes model. as required. I'lll suggest you update this too. :wave: :airplane:

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @hajnaloltyan 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- All current test cases are passing ✔
-  Model validations written correctly 👋 
-  Clean and organised code✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/models/user_spec.rb (hunk 1 above)
Comment (Davidosky007) [reply spec/models/user_spec.rb:3]: done

File: spec/models/like_spec.rb (hunk 3 above)
Comment (Davidosky007) [reply spec/models/like_spec.rb:4]: done

File: spec/models/comment_spec.rb (hunk 2 above)
Comment (Davidosky007) [reply spec/models/comment_spec.rb:3]: done

**Review by Davidosky007 — APPROVED** [review]
Hi @hajnaloltyan 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* All unit tests passed 👍 
* All models validated properly 👍 
* Well-written PR 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**